### PR TITLE
[Don't merge until Jan 9]Redirect all UToronto home pages to unified homepage

### PR DESCRIPTION
**Pull request by yuvipanda**
UToronto will have a new URL (to be determined...) that they will manage, and will act as the 'unified' home page. In https://2i2c.freshdesk.com/a/tickets/952 they have asked us to redirect anyone landing just on the home page of the hubs to this new URL. I've setup the [utoronto
branch](https://github.com/2i2c-org/default-hub-homepage/tree/utoronto) of the home pages repo to redirect to the new URL.

This just deploys that change. The new URL hasn't been provided to us yet, so the utoronto branch in the default-hub-homepage repo will need to be updated once that URL is present.

Ref https://github.com/2i2c-org/infrastructure/issues/2729

### CODE REVIEW

File: config/clusters/utoronto/common.values.yaml [hunk 1]
```diff
@@ -35,26 +38,6 @@ jupyterhub:
         funded_by:
           name: University of Toronto
           url: https://www.utoronto.ca/
-        announcements:
-          - |
-            <div>
-              <h4>ARC will be making changes to our educational JupyterHub on January 9, 2023</h4>
```
Comment (consideRatio) [inline config/clusters/utoronto/common.values.yaml:41]: "Will be making", but when they read this the changes are made - should this be revised?

File: config/clusters/utoronto/default-staging.values.yaml [hunk 2]
```diff
@@ -4,12 +4,6 @@ jupyterhub:
     tls:
       - hosts: [staging.utoronto.2i2c.cloud]
         secretName: https-auto-tls
-  custom:
-    homepage:
-      gitRepoBranch: "utoronto-staging"
-  singleuser:
-    image:
-      tag: "14320bae73a0"
```
Comment (consideRatio) [inline config/clusters/utoronto/default-staging.values.yaml:12]: Also in the other PR, but probably fine?

File: config/clusters/utoronto/common.values.yaml (hunk 1 above)
Comment (yuvipanda) [reply config/clusters/utoronto/common.values.yaml:41]: @consideRatio the home page will just not be visible at all as soon as this change is made (due to https://github.com/2i2c-org/default-hub-homepage/commit/20ea7925734f00340af4eccfc41eaacdfe894df8) so this is fine.

File: config/clusters/utoronto/default-staging.values.yaml (hunk 2 above)
Comment (yuvipanda) [reply config/clusters/utoronto/default-staging.values.yaml:12]: @consideRatio ah correct. But I've removed it from here anyway just for cleanliness.

**Pull request merged**